src/widget/weffectselector.cpp

```diff
@@ -57,15 +57,13 @@ void WEffectSelector::populate() {
                                                        width() - 2);
         addItem(elidedDisplayName, QVariant(pManifest->id()));
 
-        // NOTE(Be): Using \n instead of : as the separator does not work in
-        // QComboBox item tooltips.
-        // TODO(Be): Check if this is also the case with Qt5.
-        //: %1 = effect name; %2 = effect description
-        QString description = tr("%1: %2").arg(pManifest->name(),
-                                               pManifest->description());
+        QString name = tr("%1").arg(pManifest->name());
+        QString description = tr("%1").arg(pManifest->description());
```

Comment:
Why not this?

```suggestion
        QString name = tr(pManifest->name());
        QString description = tr(pManifest->description());
```

Reply:
:facepalm: sure..

Reply:
AFAICT we dont need `tr` here, isn't the translatable string already catched in the effect manifest?
https://github.com/mixxxdj/mixxx/blob/master/src/effects/builtin/bessel8lvmixeqeffect.cpp#L15